src/rpc/blockchain.cpp

```diff
@@ -2180,6 +2180,27 @@ UniValue getscgenesisinfo(const UniValue& params, bool fHelp)
     // block hex data
     ssBlock << block;
 
+    // Retrieve sidechain version for any sidechain that published a certificate in this block
+    std::vector<sSidechainVersion_tag> vSidechainVersion;
```

Comment:
Cosmetic: Since we are not using a typedef for this struct, consider removing the '_tag' in the name, for instance: sSidechainVersion_tag -> ScVersionInfo

Reply:
Fixed with commit [fd2808f37f8ca4972db628673de72ba0899b0ec9](https://github.com/HorizenOfficial/zen/pull/437/commits/fd2808f37f8ca4972db628673de72ba0899b0ec9).